### ISSUE TITLE
Bring across updates to javascript-detect-element-resize

### ISSUE THREAD

**Issue by benjeffery** [nrako/react-component-resizable#14]
There are a few commits such as https://github.com/sdecima/javascript-detect-element-resize/commit/b058d39bfe419c9551264d7ad9757b05306e7039 which don't seem to be reflected in the code for this component. 
Is there a reason for not using that module as a dependency?

Thanks!

**Comment by nrako** [comment]
You are right good catch, it looks like `opacity: 0` can prevent a bug in chrome.
> //opacity:0 works around a chrome bug https://code.google.com/p/chromium/issues/detail?id=286360

This was committed by the PR author before I started this component but was released after https://github.com/sdecima/javascript-detect-element-resize/releases/tag/v0.5.2, so I probably never saw that.

I used [javascript-detect-element-resize](https://github.com/sdecima/javascript-detect-element-resize) has a source of information at some point but that wasn't the original source for this component [Daniel - backalleycoder.com, original blog post](http://www.backalleycoder.com/2013/03/18/cross-browser-event-based-element-resize-detection/) was. And after a quick look javascript-detect-element-resize doesn't look very suitable to be used as dependency. Moreover Dianel blog post solution changed quite a bit and I would rather try to implement his new solution which seems more elegant.

Unfortunately I don't have yet much more personal incentive to accelerate this change, and also since current implementation doesn't trigger much more issues I'm not in a hurry.

However feel free to point to more commits which you think should be included I'll happily do a quick release, even better a PR :stuck_out_tongue_closed_eyes:  :pray:

**Comment by benjeffery** [comment]
Thanks! Ahhhh, I see now that you couldn't use that directly. Makes sense.
Will try and get a PR to you at some point.
